src/query/functions/src/scalars/string.rs

```diff
@@ -139,7 +139,9 @@ pub fn register(registry: &mut FunctionRegistry) {
             |s, pad_len, pad, output, _| {
                 let pad_len = pad_len as usize;
                 if pad_len <= s.len() {
-                    output.put_slice(&s[..pad_len])
+                    output.put_slice(&s[..pad_len]);
```

Comment:
If the `pad_len` is greater than the length of `s` and the `pad` is empty, we should return an error message because it is not possible to fill the string to the `pad_len` with an empty string.

Reply:
Logically speacking, you are right. It's ok for me. Currently, I have adopted the same behavior as Postgres. I'll fix it.

Reply:
@b41sh PTAL.